x-pack/auditbeat/tests/system/test_metricsets.py

```diff
@@ -57,6 +57,7 @@ def test_metricset_socket(self):
         self.check_metricset("system", "socket", COMMON_FIELDS + fields, warnings_allowed=True)
 
     @unittest.skipUnless(sys.platform == "linux2", "Only implemented for Linux")
+    @unittest.skip("Test is failing in CI")
```

Comment:
Do you have a github issue already to track this? we normally mention it in the skip line.

Reply:
added